src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -618,5 +650,30 @@ export const ComboBoxExample = {
       demo: <DuplicateOptions />,
       snippet: duplicateOptionsSnippet,
     },
+    {
+      title: 'Accessible label with aria-labelledby',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: labelledbySource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: labelledbyHtml,
+        },
+      ],
+      text: (
+        <p>
+          Sometimes it&rsquo;s preferable to label a combobox with a heading or
+          paragraph. You can easily create a unique ID for a text element using
+          the <a href="/#/utilities/html-id-generator">HTML ID generator</a>.
+          Then pass your unique ID to the{' '}
+          <EuiCode language="js">aria-labelledby</EuiCode> prop.
```

Comment:
```suggestion
          <EuiCode>aria-labelledby</EuiCode> prop.
```
I don't think the `language` prop adds anything here for a short inline EuiCode